cpp/binary/pymod_modmesh/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if (CLANG_TIDY_EXE AND USE_CLANG_TIDY)
 endif ()
 
 execute_process(
-    COMMAND python3-config --extension-suffix
+    COMMAND $(dirname $(realpath $(which python)))/python3-config --extension-suffix
```

Comment:
I'd be curious whether or not windows accepts it.

Reply:
I am wondering too. I am thinking if you know someone interested in testing this?

Reply:
It actually passes CI ... well, interesting.

But it does not hurt to use cmake native helpers: https://cmake.org/cmake/help/v3.0/command/get_filename_component.html

Reply:
Thanks for the tip. Let me check.